nav2_behavior_tree/CMakeLists.txt

```diff
@@ -223,8 +223,8 @@ install(DIRECTORY include/
   DESTINATION include/
 )
 
-install(DIRECTORY test/utils/
-        DESTINATION include/utils/
+install(DIRECTORY test/include/
```

Comment:
Revert, there's no reason to have an `include` inside of the `test` directory. Utils is correct.

Reply:
Actually, if you consider the include path in src, it is `package_name/src/include/package_name/header.hpp`.
Which installs to
- `ros2_workspace_dir/install/package_name/package_name/header.hpp` if `--merge-install` is used with colcon build.
- `ros2_workspace_dir/install/package_name/include/package_name/header.hpp` if `--merge-install` is absent.

Therefore, by adding an include directory in test, we ensure that the utils package installs to
- `install/nav2_behavior_tree/utils` instead of `install/utils` when `--merge-install` is used
- `install/nav2_behavior_tree/include/nav2_behavior_tree/utils` instead of `install/nav2_behavior_tree/include/utils/` when `--merge-install` is absent

Reply:
This directory schema seems nonsensical to me and I see no other examples (e.g. rclcpp, moveit2) of this structure being used. The install rule you simply update from `utils` to `include`, so there should be little difference here if we just name it `utils`. I don't see why it wouldn't be possible to still have it appear in `install/nav2_behavior_tree/test/utils` if we set the destination in the install rule. I don't see why 

> install/nav2_behavior_tree/include/nav2_behavior_tree/utils instead of install/nav2_behavior_tree/include/utils/ when --merge-install is absent

is preferable.